src/cpp/core/include/core/json/JsonRpc.hpp

```diff
@@ -287,17 +289,17 @@ core::Error readObject(const Object& object,
                        const std::string& name, 
                        T* pValue)
 {
-   Object::iterator it = object.find(name);
+   Object::Iterator it = object.find(name);
```

Comment:
For types that are designed to match the interface of something in `std`, I think it's okay to continue using the same, lowercase name. That is, classes providing iterators can still call them `iterator`, unless they don't satisfy the default `iterator` contract for some reason.

Reply:
I totally understand what you're saying, but I was going for a really strictly consistent naming style for the SDK.

Reply:
Fair enough!